app/controllers/filetypes_controller.rb

```diff
@@ -0,0 +1,27 @@
+class FiletypesController < ApplicationController
+  def create
+    render json: filetype_json
+  rescue Hets::HetsError
+    render json: {status: 415, message: 'Media Type not supported'},
+           status: 415
+  end
+
+  protected
+
+  def filetype
+    @filetype ||= Hets::FiletypeCaller.new(HetsInstance.choose).
+      call(params[:iri])
+  end
+
+  def filetype_json
```

Comment:
I believe i've seen this on a few projects. If it however is not a convention, it should now be an ontohub convention to name methods in such a way, don't you think?